ote/src/cljs/ote/views/route/route_list.cljs

```diff
@@ -45,91 +51,152 @@
+                              (or (= :schedule sub-type) false))
+                            operator-services)]
     [:div
-     [list-header/header
-      app
-      (tr [:route-list-page :header-route-list])
-      [common/tooltip
-       {:text (if (empty? operators)
-                (tr [:route-list-page :create-new-route-disabled])
-                (tr [:buttons :add-new-route]))}
-       [ui/raised-button {:label    (tr [:buttons :add-new-route])
-                          :on-click #(do
-                                       (.preventDefault %)
-                                       (e! (route-list/->CreateNewRoute)))
-                          :primary  true
-                          :disabled (empty? operators)
-                          :icon     (ic/content-add)}]]
-      [t-operator-sel/transport-operator-selection e! app]
-      [buttons/open-link
-       (tr [:route-list-page :link-to-help-pdf-url])
-       (tr [:route-list-page :link-to-help-pdf])]]
+     [page/page-controls "" (tr [:route-list-page :header-route-list])
+      [:div
+       [:h4 {:style {:margin "0"}}
+        (tr [:field-labels :select-transport-operator])]
+       [form-fields/field
+        {:element-id "select-operator-at-searoute-frontpage"
+         :name :select-transport-operator
+         :type :selection
+         :show-option #(::t-operator/name %)
+         :update! #(e! (to/->SelectOperator %))
+         :options (mapv to/take-operator-api-keys (mapv :transport-operator operators))
+         :auto-width? true
+         :class-name "mui-select-button"}
+        (to/take-operator-api-keys operator)]]]
 
-     (when routes
-       [list-routes e! routes])
-     (when routes
-       (let [loc (.-location js/document)
-             url (str (.-protocol loc) "//" (.-host loc) (.-pathname loc)
-                      "export/gtfs/" (::t-operator/id operator))]
-         [:span
-          [:br]
-          [common/help
-           [:span
-            [:div (tr [:route-list-page :route-list-active-routes])
-             [common/linkify url (tr [:route-list-page :route-list-gtfs-zip-file])]]
-            [:div {:style {:width "100%"}}
-             (tr [:route-list-page :route-list-copy-link])
-             [common/copy-to-clipboard url]]]]]))]))
+     [:div.container
+      [:h2 (get-in app [:transport-operator :ote.db.transport-operator/name])]
+      [:a (merge {:href (str "#/new-route/")
+                  :id "new-route-button"
+                  :on-click #(do
+                               (.preventDefault %)
+                               (e! (route-list/->CreateNewRoute)))}
+                 (stylefy/use-style style-buttons/primary-button))
+       (tr [:buttons :add-new-route])]
+      [list-routes e! public-routes draft-routes]
+      (when schedule-services
+        [link-services-to-routes e! schedule-services services-with-route (not (empty? public-routes))])
+      (if (not (empty? public-routes))
```

Comment:
Logical conditions with minimum amount of inversion/complemention operators are more clear, so generally 
``` if (not A) B C``` ===>  ```if A B C```